Makefile

```diff
@@ -27,85 +30,26 @@ pip-check:
 # Build and run production docker container
 
 build-prod:
-	docker build --file=Dockerfile . --tag=openslides-backend
+	docker build . --tag=openslides-backend
 
-run-prod:
+run-prod: | build-prod
 	docker run --interactive --tty \
 	--publish 9002:9002 --publish 9003:9003 --rm openslides-backend
 
 
-# Build and run development and testing docker container
-
-build-dev:
```

Comment:
We need `build-dev`. It is required in the big setup.

Reply:
Please explain what the "big setup" is. I thought we use everything together with docker compose using the production Dockerfile and not the development version (as we do in the new systems tests here concerning the datastore).

Reply:
The dev-mode in the main repo required `build-dev`.